app/main.py

```diff
@@ -1,13 +1,20 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
```

Comment:
After opening the parentheses, write each the key on a new line, including the first one

Reply:
Done